SparkPost/2web2ui
# [AC-1735] (UI) Change the format of billing cycle dates to use UTC format

[AC-1735] (UI) Add the offset of +08:00 to billing cycle dates

Note - we had to change this because api started returning the utc date with time T00:00

### What Changed
 - Add the offset of +08:00 to billing cycle dates for DashboardPage, DashboardPageV2 and UsageReport

### How To Test
 - Check if the dates match the month.start and month.end returned by GET /account?include=usage

### To Do


[AC-1735]: https://sparkpost.atlassian.net/browse/AC-1735
